src/providers/export/exportProvider.ts

```diff
@@ -82,11 +94,12 @@ export abstract class ExportProvider<TOptions> implements IExportProvider {
                 break;
             case ExportAssetState.All:
             default:
+                getAssetsFunc = getAllAssets;
                 predicate = () => true;
                 break;
         }
 
-        return await _.values(this.project.assets)
+        return (await getAssetsFunc())
             .filter((asset) => asset.type !== AssetType.Video)
```

Comment:
so we are still ignoring videos in "All asset" export?

Reply:
we ignore `AssetType.Video` for all exports.  All `Visited` or `Tagged` Video Frames are exported but we don't export the container Video asset itself.  Hope that clears up any confusion.